graphstorm-processing/LICENSE

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
I don't think we need a separate LICENSE file.

Reply:
It might be needed if the packaging authority requires such a file [to upload the library to PyPI](https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-a-license). 

I'll try removing this and see if it creates any issues with PyPI.